en/micro-integrator/docs/develop/WSO2-Integration-Studio.md

```diff
@@ -1,100 +1,128 @@
 # Quick Tour - WSO2 Integration Studio
 
-WSO2 Integration Studio is a development environment used to design your
-integration scenarios and develop them. This can be used to develop
-services, features and artifacts as well as manage their links and
-dependencies through a simplified graphical editor.
+WSO2 Integration Studio is your development environment for designing, developing, debugging, and testing integration solutions. As an integration developer, you can execute all the phases of the development lifecycle using this tool. When your integration solutions are production-ready, you can easily push the artifacts to your continuous integration/continuous deployment pipeline.
 
-## Workbench
+## Getting Started
 
-When you start using WSO2 Integration Studio, one of the first decisions
-you have to make is where to store your projects and files. This
-location is called the workspace. The Workbench is essentially different
-windows and editors that can be used for various purposes related to
-your workspace. This includes, the Project Explorer, the Editor,
-Template Guide, Outline, etc. Multiple Workbench windows can be opened
-simultaneously for an enhanced and fine-tuned developer experience.
+When you open WSO2 Integration Studio, you will see the tool's workbench, which includes the **Getting Started** view, the **Project Explorer**, and the **Template Guide**. 
```

Comment:
When you first start an integration studio, you should only see the Getting Started page.  If you start Integration Studio in new workspace you can see this view.